apps/jurisdiction/models.py

```diff
@@ -85,7 +86,7 @@ class SurveyEmail(models.Model):
 
     name = models.CharField('Email label', max_length=250)
     recipients = models.TextField('List of emails', help_text='Use commas, semicolons or line breaks to separate emails. Do not enter e-mail addresses containing those special characters.')
-    email_text = models.TextField('E-mail text', help_text='This text will be displayed above the survey links.', default = 'Thank you for your participation in our survey for WorkElections.com. Please click on the link corresponding with the jurisdiction for which you would like to update information.')
+    email_text = HTMLField('E-mail text', help_text='This text will be displayed above the survey links.', default = 'Thank you for your participation in our survey for WorkElections.com. Please click on the link corresponding with the jurisdiction for which you would like to update information.')
```

Comment:
@nayelipelayo: I had a little more time today, so I updated the "email text" field to be a rich text editor, so you can add/preview links.

Here is what the editor looks like:
<img width="1169" alt="screen shot 2018-10-19 at 4 30 00 pm" src="https://user-images.githubusercontent.com/4331931/47242423-b2178b80-d3bc-11e8-8f08-975b1918aa85.png">

And here is a screenshot of the email created with the fields in that editor:
<img width="715" alt="screen shot 2018-10-19 at 4 31 17 pm" src="https://user-images.githubusercontent.com/4331931/47242452-c9ef0f80-d3bc-11e8-8274-1acba50a50ff.png">